### DISCUSSION TITLE
0.10.1

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#5820] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/0.10.1'>0.10.1</a>.</em>

0.10.1 is a minor release with lots of bug fixes and performance improvements.

Some notable new features:
- Added rendering apis that will allow Lottie to be supported.
- Added Flyouts APIs / Controls.
- Android backend which is in preview has been updated.

Find the new binaries on nuget

**Comment by grokys** [comment]
Looks like we deprecated `ContentMenu` which is a bit of a large change for a patch release. I think we'll be undoing that and releasing 0.10.2 immediately.

**Comment by DarthSpot** [comment]
I'm working on a 4K Monitor. "Suddenly" my application seems to scale to be a useful size. Is that a new feature in 10.1?

**Reply by kekekeks** [reply]
Is that on Linux? We've fixed our EDID reader.

**Reply by DarthSpot** [reply]
Yup, KDE Plasma to be precise

**Comment by grokys** [comment]
0.10.2 has now been released with an un-deprecated `ContextMenu`.
